go.mod

```diff
@@ -28,11 +28,34 @@ require (
 
 require (
```

Comment:
Why do you need a second `require` section that contains indirect dependencies? I'm almost sure that you can drop it completely.